tests/testFile.cpp

```diff
@@ -11,7 +11,7 @@ using namespace AQNWB;
 
 TEST_CASE("ElectrodeTable", "[ecephys]")
 {
-  std::string path = "/electrodes/";
+  std::string path = "/general/extracellular_ephys/electrodes/";
```

Comment:
If we make the `electrodeTablePath` a `const static` property then we can also avoid duplicating it here and the other places this path is defined, because it is then fixed at compile time.